### PR TITLE
Mvorisek fix travis build

### PR DESCRIPTION
**Pull request by romaninsh**
Superseeds #931 by creating local branch. Thanks to @mvorisek

### CODE REVIEW

**Comment by mvorisek** [comment]
@romaninsh GH CI seems to be broken even more now... Why not to merge the TravisCI fix and solve GH CI separately?

**Comment by mvorisek** [comment]
UPDATE: GH Actions seems to be fixed by itself, see https://github.com/atk4/ui/issues/933

**Comment by mvorisek** [comment]
@romaninsh Close this, fixed by PR #931

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -35,15 +35,15 @@ jobs:
...
-        run: |
-          mkdir -p coverage
-          apk add npm
-          npm install -g less clean-css uglify-js
-          mysql -uroot -ppassword -h mysql -e 'CREATE DATABASE atk4;'
-          mysql -uroot -ppassword -h mysql atk4 < demos/atk4.sql
-
-          mv demos/db.github.php demos/db.php
-          (cd js; npm install; npm run build)
+        run:
```
Comment (mvorisek) [inline .github/workflows/unit-tests.yml]: This newly breaks GH tests. The Travis is already fixed by PR #931. This PR should be completely closed as it does not provide anything at this stage.

**Comment by mvorisek** [comment]
@DarkSide666 not ok, see comment, close instead

**Comment by DarkSide666** [comment]
Yes you're right. Thanks!

**Pull request closed without merging**